src/features/virtualized-table-renderers/__tests__/__snapshots__/draggableRowRenderer.test.js.snap

```diff
@@ -0,0 +1,29 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`features/virtualized-table-renderers/draggableRowRenderer should wrap row in Draggable and set correct props 1`] = `
```

Comment:
I think for snapshots like this, we should consider regular `expect` checks. There is no DOM structure here in the snapshot.

Reply:
Agreed. I actually have some regular checks on that test. Let me see if I can replace the snapshot with some more `expect`s.